frontend/src/components/TileContainer.tsx

```diff
@@ -12,22 +13,43 @@ const TileContainer = (props: TileInterface): ReactElement => {
   const { tile } = props;
 
   const [currentTileState, setTileState] = useState<TileState>(TileState.ACTIVE);
+  const [tileRotation, setTileRotation] = useState<Rotation>(tile.rotation);
 
   function handleActiveTileClick(event: MouseEvent<HTMLImageElement>): void {
     event.preventDefault();
     setTileState(TileState.TAKEN);
   }
+
+  function handleTileRotation(event: MouseEvent<HTMLImageElement>): void {
+    event.preventDefault();
+    setTileRotation(tile.rotation);
+  }
+
   return (
-    <div className="relative flex " onClick={currentTileState === TileState.ACTIVE ? handleActiveTileClick : undefined}>
+    // eslint-disable-next-line jsx-a11y/no-static-element-interactions
+    <div
+      className="relative flex "
+      onClick={currentTileState === TileState.ACTIVE ? handleActiveTileClick : handleTileRotation}
+    >
+      <ArrowButton tile={tile} direction="right" />
```

Comment:
please create separate component with
```jsx
<ArrowButton right/>
<TileContainer> 
<ArrowButton left/> 
```
we do not want to modify current Tile Container with arrows as it will be used in many places of the app